lib/vanagon/driver.rb

```diff
@@ -13,17 +13,19 @@ class Driver
     attr_accessor :platform, :project, :target, :workdir, :verbose, :preserve
     attr_accessor :timeout, :retry_count
 
-    def initialize(platform, project, options = { :configdir => nil, :target => nil, :engine => nil, :components => nil, :skipcheck => false, :verbose => false, :preserve => false }) # rubocop:disable Metrics/AbcSize
+    def initialize(platform, project, options = { :configdir => nil, :target => nil, :engine => nil, :components => nil, :skipcheck => false, :verbose => false, :preserve => false, :only_build => nil }) # rubocop:disable Metrics/AbcSize
```

Comment:
This is a good candidate to refactor -- that options hash is
![bananas-smaller](https://cloud.githubusercontent.com/assets/344926/18687834/b7dfb114-7f36-11e6-9e52-8e936a4c7ded.jpg)

probably not important in this PR but SOON because this is starting to get out of control. For now, consider moving the `options` hash itself into an external value, and then loading it in:

```ruby

# This is a naive implementation since some values are
# booleans and some are nil. But it'll demonstrate a
# viable direction we can head towards in the shorter term.
# Rather than leave the options that are nil undefined for
# brevity, by specifying them here we are at least communicating
# what options we would expect #new to receive.
DEFAULT_OPTIONS = {
  configdir: nil,
  target: nil,
  engine: nil,
  components: nil,
  skipcheck: false,
  verbose: false,
  preserve: false,
  only_build: nil
}
def initialize(platform, project, options = DEFAULT_OPTIONS) # rubocop:disable Metrics/AbcSize
```

This still has the same shortcomings, like default values not being initialized for each key in the `options` hash when any values are passed (options hashes are all-or-nothing) but it at least keeps the options hash readable.